.env.sample

```
@@ -1,3 +1,20 @@
+## PROJECT CONFIGURATION
+NUXT_PUBLIC_APP_ENV="production"
+NUXT_PUBLIC_BASE_UR="/api"
```

dotenv-linter (3.3.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)